core/common/src/RealityDataAccessProps.ts

```diff
@@ -2,10 +2,11 @@
 * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
-/** @packageDocumentation
- * @module RealityData
- */
 
+/**
+ * @docs-group-description RealityData
```

Comment:
Please put the `@docs-group-description` into core-common.ts and use `@packageDocumentation` in source files.
Ideally, replace "ProjectWise Context Share" with something more general, e.g., "reality data", since not all reality data need come from ProjectWise.